### PR TITLE
Lionel Smith-Gordon SecurityFinancingEnumerator

### PR DESCRIPTION
**Pull request by regnosys-prod-user**

### CODE REVIEW

File: docs/securities-lending.md [hunk 1]
```diff
@@ -45,50 +45,54 @@ In a cash loan, the lender lends the borrower the requested securities,
...
+other terms payable using an `InterestRatePayout`.
 
 The `collateralType` under `collateralProvisions` should be set to "*Cash*" to 
 define this as a cash collateralised loan. These types can be found under the 
 `economicTerms->collateral` type.
 
-An example showing the location of the payouts for the cash collateral and the 
-security being lent is provided below:
+An example showing the location of the 
+security being lent and the payouts for the cash collateral and is provided below:
```
Comment (chrisisla) [inline docs/securities-lending.md:63]: second "and" can be removed

File: docs/securities-lending.md [hunk 2]
```diff
@@ -99,10 +103,15 @@ In a non-cash loan, the lender lends the borrower the requested securities,
...
 products. 
 
-Loans against non-cash collateral are formed as a `NonTransferableProduct` that 
-includes a `SettlementPayout` containing with an `Asset` for the non-cash collateral, held under 
-`economicTerms->collateral`, and another `AssetPayout` for the security being 
-lent, held under `economicTerms->payout`.
+The security that is being lent is modelled the same way as the cash case with
+an `underlier` in the main `AssetPayout`.
+
+The non-cash collateral is also still modelled with a one or more `TransferableProduct`
```
Comment (chrisisla) [inline docs/securities-lending.md:109]: can remove the "a"

**Review by chrisisla — APPROVED** [review]
Couple of minor corrections to the documentation but otherwise this all looks great 👍

**Comment by PayalKhanna** [comment]
@chrisisla I have updated the text as per you comments can you please rereview

**Review by chrisisla — APPROVED** [review]
Changes look good, thanks @PayalKhanna

**Pull request merged**